### DISCUSSION TITLE
Curse of Strahd: Barovian Calendar

### DISCUSSION THREAD

**Discussion by cfloyd18** [vigoren/foundryvtt-simple-calendar#556] [Show and tell]
Hey... I think I got one working that you might be able to incorporate? I've used a few different ideas that I found online for the names of the months. But all in all.. I think it works out well. Keeps with the current year, cycles, changes of the seasons, etc.  If you want, I can try to send you the export of my configuration? 
![image](https://github.com/vigoren/foundryvtt-simple-calendar/assets/43119151/b119d1c4-ba65-4657-958d-edb2d9a44cc0)

**Comment by vigoren** [comment]
Yeah for sure! If you export the calendar configuration and share that I can include it as a default calendar option.

**Comment by cfloyd18** [comment]
I'll send it over when I get home! I tried to share it in the GitHub message originally, but it told me it was the wrong format so wouldn't attach it. I don't believe I'm missing anything. Ran into another person asking for this type of map on the Foundry VTT (specifically asking about setting up a 28 day calendar). So I had already done this so was assuming they were doing Barovia, or something close to it. Turns out I was right. He's loading it up now so hopefully I'll get feedback on if it was set up correctly to his needs. I did take some ideas from a few different postings for the names of the months, as there was some conflicting information/ideas out there. But the Moon cycle seemed consistent if I recall. The Full Moon phase ends one month and begins then next, while the New moon phase is in the middle of the month. Either way, I'm glad to help. 
v/r
Chris Floyd    On Tuesday, October 10, 2023 at 11:44:19 AM EDT, vigoren ***@***.***> wrote:  
 
 


Yeah for sure! If you export the calendar configuration and share that I can include it as a default calendar option.
